app/views/lessons/show.html.erb

```diff
@@ -49,6 +49,12 @@
       <% end %>
     </div>
 
+    <%= link_to 'Improve this lesson on Github',
```

Comment:
@KevinMulhern @I3uckwheat Feel free to suggest an alternative here.

Reply:
Looks good to me @arku, we could add a github icon to the start of the line. But thats certainly not a blocker.

Reply:
Done.